From 3fa237b3afabc293e563292b8d89265a871626ad Mon Sep 17 00:00:00 2001
From: Martin Kelly <mkelly@xevo.com>
Date: Mon, 22 May 2017 17:00:05 -0700
Subject: [PATCH] add #include <sys/sysmacros.h>

In newer glibc versions, the definition for major() has been moved to
sys/sysmacros.h, and using the older version in <sys/types.h> has been
deprecated. So, add an include for <sys/sysmacros.h>.

Upstream-Status: Pending

Signed-off-by: Martin Kelly <mkelly@xevo.com>
---
 open-vm-tools/lib/wiper/wiperPosix.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/open-vm-tools/lib/wiper/wiperPosix.c b/open-vm-tools/lib/wiper/wiperPosix.c
index bd542410..ccf06293 100644
--- a/open-vm-tools/lib/wiper/wiperPosix.c
+++ b/open-vm-tools/lib/wiper/wiperPosix.c
@@ -43,6 +43,9 @@
 #  include <libgen.h>
 # endif /* __FreeBSD_version >= 500000 */
 #endif
+#if defined(__linux__)
+#include <sys/sysmacros.h>
+#endif
 #include <unistd.h>
 
 #include "vmware.h"
